NewsApp/Business/View/BusinessViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  BusinessViewController.swift
+//  NewsApp
+//
+//  Created by Viki on 05/12/2023.
+//
+
+import UIKit
+import SnapKit
+
+class BusinessViewController: UIViewController {
+    
+    // MARK: - GUI Variables
+    
+    private lazy var collectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        
+        
+        
+        layout.minimumLineSpacing = 5
+        layout.minimumInteritemSpacing = 5
+        layout.sectionInset = UIEdgeInsets(top: 20, left: 20, bottom: 20, right: 20)
+        
+        
+        let collectionView =  UICollectionView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height), collectionViewLayout: layout)
+        
+        collectionView.dataSource = self
+        collectionView.delegate = self
+        collectionView.backgroundColor = .white
+        
+        return collectionView
+    }()
+    
+    // MARK: - Properties
+    
+    private var viewModel: BusinessViewModelProtocol
+    
+    // MARK: - Life cycle
+    
+    init(viewModel: BusinessViewModelProtocol) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+        self.setupViewModel()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupUI()
+        collectionView.register(GeneralCollectionViewCell.self, forCellWithReuseIdentifier: "GeneralCollectionViewCell")
+        collectionView.register(DetailsCollectionViewCell.self, forCellWithReuseIdentifier: "DetailsCollectionViewCell")
+        
+        viewModel.loadData()
+    }
+    
+    // MARK: - Methods
+    
+    private func setupViewModel() {
+        viewModel.reloadData = { [weak self] in
+            self?.collectionView.reloadData()
+            
+        }
+        
+        viewModel.reloadCell = { [weak self] row in
+            self?.collectionView.reloadItems(at: [IndexPath(row: row, section: 0)])
```

Comment:
section can not be always 0 because you have 2 sections in this Business collection view.